### ISSUE TITLE
HASSIO Plugin Ideas

### ISSUE THREAD

**Issue by roblandry** [Aircoookie/WLED#452]
It would be great if there were a plugin much like [TasmoAdmin](https://github.com/hassio-addons/addon-tasmoadmin), but for wled. This could allow updating multiple wleds at the same time, as well as centralized control.

**Comment by r100gs** [comment]
Yes, that would be great! And than a possibility to control all the leds by sound

**Comment by pruwait** [comment]
+

**Comment by yuvalabou** [comment]
+1

**Comment by Aircoookie** [comment]
I am not in a position to add something like this, but I would be happy to collaborate with someone who whould write such a plugin by making changes to WLED that might be needed for such a plugin to work (although there aren't many I suppose)

**Comment by austwhite** [comment]
I think Frenck has written a nice integration for HA.  Might be worth a look.

**Comment by softhack007** [comment]
Closing, as there was no activity on this enhancement proposal since 3 years.

Meanwhile HA has something for controlling WLED instances, and control by audio is also possible (thanks to a bunch of brilliant developers 😉 )
